### PR TITLE
ROX-25354: PR GA requirements document

### PR DESCRIPTION
**Pull request by 0x656b694d**
### Description

Doc.

### User-facing documentation

(*must be* 2 items and both *must be* checked)

- [x] CHANGELOG update is not needed
- [x] Documentation PR is created and linked above

### Testing

- [x] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [x] contributed **no automated tests**
  

#### How I validated my change

Peer review.

### CODE REVIEW

File: PR_GA.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# General Availability (GA) Requirements for a Pull Request
+
+1. **Code Quality**
+   - The PR must pass all automated tests (unit, integration, end-to-end) without any failures.
+   - The code must follow the project's coding standards and guidelines.
```
Comment (rhybrillou) [inline PR_GA.md:5]: Do we have any document for these standards ?
Some of them are enforced by linters. Style checks must also pass.

**Review by rhybrillou — COMMENTED** [review]
Can we make sure existing documents backing requirements are also linked here (e.g. coding standard) ?

File: PR_GA.md (hunk 1 above)
Comment (0x656b694d) [reply PR_GA.md:5]: I don't think we've got something global, but there is at least the error reporting guidelines on Confluence.
Not sure how to link it here. We might better move such guidelines to the repo.
Style checks are covered by the next item: static code analysis tools.

Comment (rhybrillou) [reply PR_GA.md:5]: I'd rather not link confluence (internal) documentation from within the code repositories.
I also think if something should be public and referenced from the code repository, it should be in a public code repository.

**Review by rhybrillou — APPROVED** [review]

**Pull request merged**